config/deploy.rb

```diff
@@ -78,7 +78,7 @@
       within("#{current_path}/pywb") do
         # Make sure python executables are on the PATH
         with(path: '$HOME/.local/bin:$PATH') do
-          execute :pip3, :install, 'poetry'
+          execute :pip3, :install, '--upgrade', 'poetry'
```

Comment:
Get the latest version of poetry. I guess we could pin it, but maybe it's better to let it float and notice when we run into a problem?